tests/e2e/tests/MapComponent/MapComponentVisible.ts

```diff
@@ -32,7 +33,11 @@ export default (
         countryName: NoTriggerDataSet.CountryName,
       });
       await map.mapComponentIsVisible();
-      await map.breadCrumbViewIsVisible({ nationalView: true });
+
+      // PROTOTYPE EXAMPLE of disaster-type specific assertion
+      if (disasterType === DisasterTypeEnum.Floods) {
```

Comment:
Are the breadcrumbs only used in **flood**?

We should use these features consistently for all country hazards. We can use _conditional_ tests as a crutch until we refactor these exceptions out of the portal. In the long term, such branching of logic will only get more complex thus making IBF (and its tests) unmanageable.